Tasks/DeployVisualStudioTestAgent/TestAgentConfiguration.ps1

```diff
@@ -802,12 +807,14 @@ function InvokeTestAgentConfigExe([string[]] $Arguments, [string] $Version, [Sys
     $ExeName = "TestAgentConfig.exe"
     if (-not (Test-IsAdmin))
     {
+        Write-Host "##vso[task.logissue type=error;code=001005;]"
         throw "You need to be an Administrator to run this tool."
     }
 
     $vsRoot = Locate-TestVersionAndVsRoot($Version)
     if ([string]::IsNullOrWhiteSpace($vsRoot))
     {
+        Write-Host "##vso[task.logissue type=error;code=001003;]"
```

Comment:
have tried to keep similar ones with the same error code. however this is remotely run so removed all write-hosts from these ps1 files